portfolio/src/main/webapp/library.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 Google LLC
```

Comment:
Typically the copyright date is set to the year when the code was first written (so 2020 in this case).